content/tutorials/map_algebra/basic_map_algebra.qmd

```diff
@@ -0,0 +1,511 @@
+---
+title: "Basics of map algebra"
```

Comment:
I think I would highlight the terrain theme in the title, because it's very prominent in the tutorial